Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,37 @@
-const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+
+/*const images = [
+  "./assets/cute-cat-1.png",
+  "./assets/cute-cat-2.jpg",
+  "./assets/cute-cat-3.jpg",
+  "./assets/cute-cat-4.jpg",
+  "./assets/cute-cat-5.jpg",
 ];
+*/
+
+// Write your code here
+
+var number_img = 1;
+var total_images = 5;
+
+function img_change_add() {
+  var img = document.getElementById("img");
+  
 
+   if (number_img < total_images) {
+     number_img += 1;
+   } else {
+     number_img = 1; 
+   }
+  img.src = "/Sprint-3/slideshow/assets/cute-cat-" + number_img + ".jpg";
```

Comment:
What if you were asked to add images to the slide show (without downloading them to your website)? Can you figure out a more flexible solution so that the slide show app can work for any number of images (regardless whether they are located on your website or located on other websites)?